testData/solutions/win32.json5

```diff
@@ -3002,6 +3002,613 @@
         ]
     },
 
+    "com.microsoft.windows.internationalization": {
+        "name": "Windows Cursors",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.registrySettingsHandler",
+                "options": {
+                    "hKey": "HKEY_CURRENT_USER",
+                    "path": "Control Panel\\International",
+                    "dataTypes": {
+                        "iFirstDayOfWeek": "REG_SZ",
+                        "sShortDate": "REG_SZ",
+                        "sLongDate": "REG_SZ",
+                        "sShortTime": "REG_SZ",
+                        "sTimeFormat": "REG_SZ",
+                        "sDecimal": "REG_SZ",
+                        "iDigits": "REG_SZ",
+                        "sThousand": "REG_SZ",
+                        "sGrouping": "REG_SZ",
+                        "sNegativeSign": "REG_SZ",
+                        "iNegNumber": "REG_SZ",
+                        "iLZero": "REG_SZ",
+                        "sList": "REG_SZ",
+                        "iMeasure": "REG_SZ",
+                        "sNativeDigits": "REG_SZ",
+                        "NumShape": "REG_SZ",
+                        "sCurrency": "REG_SZ",
+                        "iCurrency": "REG_SZ",
+                        "iNegCurr": "REG_SZ",
+                        "sMonDecimalSep": "REG_SZ",
+                        "iCurrDigits": "REG_SZ",
+                        "sMonThousandSep": "REG_SZ",
+                        "sMonGrouping": "REG_SZ"
+                    },
+                    "verifySettings": true,
+                    "message": {
+                        "hwnd": "HWND_BROADCAST",
+                        "msg": "WM_SETTINGCHANGE",
+                        "wparam": 0,
+                        "lparam": "intl"
+                    }
+                },
+                "supportedSettings": {
```

Comment:
It might be worth linking to the source of this information.

(was it this? https://msdn.microsoft.com/library/dd464799.aspx)

Reply:
I think this information was taken from the registry itself, seeing the changing values through the settings app, by pure observation.